src/main/java/baritone/process/FarmProcess.java

```diff
@@ -83,7 +87,8 @@
             Items.CARROT,
             Items.NETHER_WART,
             Items.REEDS,
-            Item.getItemFromBlock(Blocks.CACTUS)
+            Item.getItemFromBlock(Blocks.CACTUS)//,
+            //Item.SWEET_BERRIES
```

Comment:
what

Reply:
[Sweet berries was added in 1.14](https://minecraft.gamepedia.com/Sweet_Berries)